### PR TITLE
[chore] Move AWS ECS, EKS and Log attributes to the registry

### PR DESCRIPTION
**Pull request by joaopgrassi**
Related to https://github.com/open-telemetry/semantic-conventions/issues/407

## Changes

Move AWS ECS, EKS and Log to the attributes registry

## Merge requirement checklist

* [x] [CONTRIBUTING.md](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md) guidelines followed.
* [ ] Change log entry added, according to the guidelines in [When to add a changelog entry](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#when-to-add-a-changelog-entry).
  * If your PR does not need a change log, start the PR title with `[chore]`
* [ ] [schema-next.yaml](https://github.com/open-telemetry/semantic-conventions/blob/main/schema-next.yaml) updated with changes to existing conventions.

### CODE REVIEW

**Review by lmolkova — APPROVED** [review]

**Comment by AlexanderWert** [comment]
Let's have a single `aws` file in the registry. See related comment in https://github.com/open-telemetry/semantic-conventions/pull/850/files#r1542415558.

**Review by pyohannes — APPROVED** [review]

**Review by arminru — APPROVED** [review]

**Comment by joaopgrassi** [comment]
> Let's have a single `aws` file in the registry. See related comment in [#850 (files)](https://github.com/open-telemetry/semantic-conventions/pull/850/files#r1542415558).

@AlexanderWert fixed!

File: model/registry/aws.yaml [hunk 1]
```diff
@@ -240,3 +240,117 @@ groups:
                   "WriteCapacityUnits": number
                 }
               }'
+  - id: registry.aws.ecs
```
Comment (trisch-me) [inline model/registry/aws.yaml:243]: I think this is similar to the https://github.com/open-telemetry/semantic-conventions/pull/872#discussion_r1549913202

**Review by arminru — APPROVED** [review]

**Review by lmolkova — APPROVED** [review]

**Review by AlexanderWert — APPROVED** [review]

File: model/registry/aws.yaml (hunk 1 above)
Comment (joaopgrassi) [reply model/registry/aws.yaml:243]: I created an issue to keep the discussion going: https://github.com/open-telemetry/semantic-conventions/issues/887

**Pull request merged**